release_notes/367-release-notes.adoc

```diff
@@ -0,0 +1,99 @@
+[id="367-release-notes"]
+= Red Hat Advanced Cluster Security for Kubernetes 3.67
+include::modules/common-attributes.adoc[]
+:context: 367-release-notes
+
+toc::[]
+
+{product-title} (RHACS) 3.67 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+*Release date:* November 29, 2021
+
+[id="new-features"]
+== New features
+
+[id="openshift-dedicated-support"]
+=== OpenShift Dedicated support
+{product-title} 3.67 is thoroughly tested and supported on OpenShift Dedicated on Amazon Web Services and Google Cloud.
+
+[id="use-ocp-oauth-as-identity-provider"]
+=== Use {ocp} OAuth server as an identity provider
+If you are using {product-title} with {ocp}, you can now configure the built-in {ocp} OAuth server as an identity provider for {product-title}. For more details, see xref:../operating/manage-user-access/configure-ocp-oauth.adoc[Configuring {ocp} Oauth server as an identity provider in {product-title}].
+
+[id="enhancements-for-ci-outputs"]
+=== Enhancements for CI outputs
+Red Hat has improved the usability of {product-title} CI integrations. CI outputs now show additional detailed information about the vulnerabilities and the security policies responsible for broken builds.
+
+[id="new-runtimeclass-policy-criteria"]
+=== Runtime Class policy criteria
+Users can now use ACS to define the container runtime configuration that may be used to run a pods containers using the *Runtime Class* policy criteria.
+
+[id="important-bug-fixes"]
+== Important bug fixes
+
+* *ROX-7815*: Previously, when using ACS with the Compliance Operator integration, ACS did not respect or populate Compliance Operator TailoredProfiles. We’ve fixed this issue.
```

Comment:
s/ACS/RHACS (I think :slightly_smiling_face:)
Also, should TailoredProfiles be in backticks or else made into a normal phrase?